src/core/rotation_ast.py

```
@@ -0,0 +1,273 @@
+            for c in subs:
+                n = _normalize_cond(c, warnings)
+                if n is not None:
+                    cleaned.append(n)
+            if not cleaned:
+                warnings.append("all 内无有效条件，已丢弃整个 all")
+                return None
+            return {"all": cleaned}
+        if "any" in cond:
+            subs = cond["any"]
+            if not isinstance(subs, list):
+                warnings.append(f"any 非列表，已视为 []: {subs!r}")
```

Ruff (0.16.0)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)